stan/math/prim/prob/neg_binomial_lpmf.hpp

```diff
@@ -15,6 +15,12 @@
 namespace stan {
 namespace math {
 
+namespace internal {
+// Exposing to let me use in tests
```

Comment:
Comments shouldn't be first person.  I'd suggest just eliminating line 19.

Line 20 should say what the value is for---in this case, the threshold beyond which a Poisson approximation is used.  It's also OK to say why it's 1e10, but I couldn't follow the logic of why it's set to 1e10.

Reply:
@bob-carpenter, would it be ok if L20 said something like:
``` // Set to 1e10 based on empirical evaluation. At 1e9, the approximation isn't close enough```